src/host/settings.hpp

```diff
@@ -213,6 +222,7 @@ class Settings
     std::wstring _LaunchFaceName;
     bool _fAllowAltF4Close;
     DWORD _dwVirtTermLevel;
+    SettingsTextMeasurementMode _textMeasurement = SettingsTextMeasurementMode::Graphemes;

```

Comment:
Should we consider having a different default for inbox (in 3 years?)

Reply:
My opinion: I consider it unlikely that this will break any applications. After all, what use would printing a zero-width character have in conhost at the moment? In particular if we consider for how long our `ExtTextOutW` support was broken... I think this will be a non-issue.

If it does become an issue, we do have the registry key someone could set.